.github/workflows/publish_docs.yaml

```diff
@@ -0,0 +1,53 @@
+name: Publish docs
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:9.6
+      redis:
+        image: redis
+    env:
+      TEST: 'docs'
+    steps:
+      - uses: actions/checkout@v2
+      - name: GitHub Environment Variables Action
+        uses: FranzDiebold/github-env-vars-action@v1.2.1
```

Comment:
cool!

Reply:
I would prefer that for 3rd party actions we get in the habit of using commit IDs instead of tags, since tags are mutable.  Actions developed by github (actions/ prefix) are probably safe.